### PR TITLE
Allow modifying of existing orders (#426)

### PR DESCRIPTION
**Pull request by peterbarrow**
Any orders that were already placed that are not part of the LP shape can be cancelled and amended like a normal order.

### CODE REVIEW

File: specs/0038-liquidity-provision-order-type.md [hunk 1]
```diff
@@ -141,6 +141,8 @@ Liquidity providers are always allowed to amend their shape generated orders by
 
 No cancellation of orders other than by lowering commitment as per [[Liquidity Provisioning mechanics spec](./0044-lp-mechanics.md).
 
+Existing orders that are used as part of the liquidity commitment can be cancelled and amended just like normal orders on the book which are not part of liquidity provisioning.
```
Comment (tamlyn10) [inline specs/0038-liquidity-provision-order-type.md]: I think we should be clear this is about limit orders..

**Review by tamlyn10 — CHANGES_REQUESTED** [review]
one minor comment - if fixed then good to merge

File: specs/0038-liquidity-provision-order-type.md (hunk 1 above)
Comment (peterbarrow) [reply specs/0038-liquidity-provision-order-type.md]: I understood it to mean all persistent orders which includes limit and pegged orders?

Comment (barnabee) [reply specs/0038-liquidity-provision-order-type.md]: @peterbarrow yeah, I think that's correct, the only orders that can't be amended are those that are derived from the liquidity commitment, as far as I know (i.e. this should include any order that would normally be amendable)

Comment (tamlyn10) [reply specs/0038-liquidity-provision-order-type.md]: ah i thought it referred to this question.... https://vegaprotocol.slack.com/archives/CAHA5EX0F/p1606217324042000

**Comment by davidsiska-vega** [comment]
@peterbarrow I've not seen this one when dealing with the #426 so I ended up creating a different pull request. Decide which one you want to keep and delete the other one (the other one being https://github.com/vegaprotocol/product/pull/459 ). 

D

**Comment by peterbarrow** [comment]
Has been superseded, closing.

**Pull request closed without merging**